data/data/bootstrap/baremetal/README.md

```diff
@@ -22,19 +22,37 @@ Relevant files:
 * **systemd/units/keepalived.service** - systemd unit file for `keepalived`.
   This runs `keepalived.sh` to generate the proper configuration from the
   template and then runs podman to launch `keepalived`.
+* **files/usr/local/bin/fletcher8** - Script that uses the
+  [fletcher8](https://en.wikipedia.org/wiki/Fletcher%27s_checksum) algorithm to
+  generate a hash from an input string. This is used to generate VRRP ids for
```

Comment:
based on the cluster name, in order to avoid collisions with other clusters on the same network?

Reply:
nice one, I've incorporated this